ArcGISRuntimeSDKQt_CppSamples/Scenes/AddIntegratedMeshLayer/AddIntegreatedMeshLayer.qrc

```diff
@@ -0,0 +1,11 @@
+<RCC>
+    <qresource prefix="/Samples/Scenes/AddIntegreatedMeshLayer">
+        <file alias="sample.json">AddIntegreatedMeshLayer.json</file>        
+        <file>AddIntegreatedMeshLayer.qml</file>
+        <file>AddIntegreatedMeshLayer.h</file>
+        <file>AddIntegreatedMeshLayer.cpp</file>
+		<file>main.qml</file>
```

Comment:
Tabs on this line. Please convert to spaces.

Reply:
addressed